src/core/api/index.js

```diff
@@ -62,27 +63,35 @@ export function callApi({
 
   return fetch(apiURL, options)
     .then((response) => {
-      return response.clone().json()
-        .then(
-          (jsonResponse) => ({ response, jsonResponse }),
-          (error) => {
+      function makeTextResponseJson(textResponse) {
+        return { jsonResponse: { text: textResponse }, response };
+      }
+
+      if (response.headers.get('Content-Type') === 'application/json') {
```

Comment:
The [docs](https://developer.mozilla.org/en-US/docs/Web/API/Headers/get) suggest this would be an array. Although it does include a footnote we'll probably want to handle (unless universal-fetch does this for us but I would suspect not):

> [1] Prior to Firefox 52, get() only returned the first value in the specified header, with getAll() returning all values. From 52 onwards, get() now returns all values and getAll() has been deleted.

I'm also a bit curious how this would handle the server returning `content-type`, hopefully that string is case-insensitive.